sdks/python/README.md

```diff
@@ -2,18 +2,92 @@
 
 The Redactive Python SDK provides a robust and intuitive interface for interacting with the Redactive platform, enabling developers to seamlessly integrate powerful data redaction and anonymization capabilities into their Python applications.
 
-## Building the Python SDK
+## Documentation
 
```

Comment:
Is there supposed to be something under this heading?

Reply:
I compared with other open source SDK. It is should be fine.
e.g. https://github.com/stripe/stripe-python

Reply:
... that page doesn't have the heading Documentation with no content below it?? Did you mean for `Installation` to be a subheading of documentation? Right now they're both at the same level.